auth-web/src/components/auth/AddBusinessForm.vue

```diff
@@ -159,6 +159,15 @@ export default class AddBusinessForm extends Vue {
         // attempt to add business
         await this.addBusiness({ businessIdentifier: this.businessIdentifier.trim().toUpperCase(), passCode: this.passcode })
 
+        // TODO if 201 -> call legal-api to get business
```

Comment:
is the comments in right place?

Reply:
thats todo in the add business page, I am going to work on that next